### DISCUSSION TITLE
[Feature] manually select location to backup

### DISCUSSION THREAD

**Discussion by Alacard** [immich-app/immich#8964] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I use my phone to take/keep copy of receipts, in the directory where i keep my receipts, i have tagged ".nomedia" (disabled android default auto scan from scanning that directory/sub directories) to keep my photo gallery from being spammed with receipts. but i would still like to backup that directory. would be nice if there was a way to manually select folders to backup and not just rely on android default auto scan

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075
